downloader/src/main/kotlin/Main.kt

```diff
@@ -70,14 +70,6 @@ object Main {
     enum class DataEntity {
         PACKAGES,
         PROJECT;
-
```

Comment:
Commit message: SHA1 missing after "removed in ...".

Reply:
Done.

Reply:
You shouldn't rebase to not break the link, right? As far as I remember GitHub also rebases when your branch is up-to-date?

Reply:
True, good point 😞 *Maybe* they changed that by now and do a fast-forward if they can...

Reply:
Or should I rephrase as "previous commit"?

Reply:
> *Maybe* they changed that by now and do a fast-forward if they can...

No, they still always rewrite commits 😢 I just checked.